src/hotspot/share/jfr/metadata/metadata.xsd

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+===========================================================================
+(c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+===========================================================================
+-->
+
```

Comment:
If this file is otherwise identical to upstream, then no IBM copyright should be added. If it is different, please explain those differences and put the IBM copyright notice after the Oracle notice.

Reply:
Yes these two files are identical to upstream, except the original `metadata.xsd` contains a trailing whitespace. I removed the whitespace. Does that make it "different"?

Reply:
Please don't make any changes, not even to trailing whitespace.